guide/src/migration.md

```diff
@@ -110,8 +111,8 @@ let obj_ref = obj.borrow();
 ```
 
 #### Object extraction
-Now for `T: PyClass`, `&T` and `&mut T` don't have `FromPyObject` implementation.
-Instead, you can use `&PyCell`, `PyRef`, and `PyRefMut` for object extraction.
+`T: PyClass`, `&T` and `&mut T` no longer have `FromPyObject` implementations.
+Instead you should extract `&PyCell`, `PyRef`, and `PyRefMut` respectively.
```

Comment:
```suggestion
Instead you should extract `PyRef<T>` or `PyRefMut<T>`, respectively.  You can also extract `&PyCell<T>`.
```
(I don't know when you would want to extract `PyCell` when you can do the latter two.)